### ISSUE TITLE
Provide a way to quickly print the SDK versions

### ISSUE THREAD

**Issue by hmelder** [gnustep/tools-windows-msvc#40]
When opening an issue related to the Windows SDK, it might be useful to have a command which prints out the versions of all subcomponents (libobjc2, libffi, libcurl, gnustep-make, etc.)

**Comment by hmelder** [comment]
objcfetch (homage to neofetch)

**Comment by triplef** [comment]
Good idea. tools-android does something along these lines where it [writes a build.txt](https://github.com/gnustep/tools-android/blob/10d37d9256affc87170d3bce9cde7b929f1aa2f4/build.sh#L186) containing the commit hashes of all projects that were build. I never got around of doing something similar here mainly because of everything being more difficult in Batch...

**Comment by hmelder** [comment]
> everything being more difficult in Batch...

Well, then let's write it in {Objective-}C ^^
